data/presets/presets/shop/agrarian.json

```diff
@@ -0,0 +1,27 @@
+{
+    "icon": "shop",
+    "fields": [
+        "name",
+        "brand",
```

Comment:
Can you remove `brand`, otherwise looks ok...

None of our other shop presets offer this field because we don't have a good way to suggest values for it.  Users would need to just know how to create a semicolon separated list of brands sold here.